libfwupdplugin/fu-udev-device.c

```diff
@@ -697,6 +698,18 @@ fu_udev_device_get_slot_depth(FuUdevDevice *self, const gchar *subsystem)
 	return 0;
 }
 
+static void
+fu_udev_device_probe_complete(FuDevice *device)
+{
+	FuUdevDevice *self = FU_UDEV_DEVICE(device);
+	FuUdevDevicePrivate *priv = GET_PRIVATE(self);
+
+	if (priv->udev_device != NULL) {
+		priv->udev_device_cleared = TRUE;
+		g_clear_object(&priv->udev_device);
```

Comment:
how much resource are we really talking about here?  I would think it's not a worthwhile trade off.

Reply:
200kB -- about 3% of the idle RSS of fwupd on my system...

Reply:
Okay that's more than I expected. I guess it's worth it.